<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlTabWidget</class>
 <widget class="QWidget" name="ControlTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidgetControl">
     <property name="currentIndex">
      <number>6</number>
     </property>
     <widget class="BannaWidget" name="tabBanna">
      <attribute name="title">
       <string>轮播图</string>
      </attribute>
     </widget>
     <widget class="CylinderWidget" name="tabCylinder">
      <attribute name="title">
       <string> 柱状图</string>
      </attribute>
     </widget>
     <widget class="ProgressWidget" name="tabProgress">
      <attribute name="title">
       <string>进度条</string>
      </attribute>
     </widget>
     <widget class="FrameWidget" name="tabFrame">
      <attribute name="title">
       <string>面板</string>
      </attribute>
     </widget>
     <widget class="ListWidget" name="tabList">
      <attribute name="title">
       <string>列表</string>
      </attribute>
     </widget>
     <widget class="TableWidget" name="tabTableWidget">
      <attribute name="title">
       <string> 表格</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BannaWidget</class>
   <extends>QWidget</extends>
   <header location="global">maincontent/controlwidget/bannacontrol/bannawidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CylinderWidget</class>
   <extends>QWidget</extends>
   <header location="global">maincontent/controlwidget/cylindercontrol/cylinderwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProgressWidget</class>
   <extends>QWidget</extends>
   <header location="global">maincontent/controlwidget/progresscontrol/progresswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FrameWidget</class>
   <extends>QWidget</extends>
   <header location="global">maincontent/controlwidget/framecontrol/framewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QWidget</extends>
   <header location="global">maincontent/controlwidget/tablecontrol/tablewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OpenglWidget</class>
   <extends>QWidget</extends>
   <header location="global">maincontent/controlwidget/openglcontrol/openglwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProcessWidget</class>
   <extends>QWidget</extends>
   <header>maincontent/controlwidget/processcontrol/processwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListWidget</class>
   <extends>QWidget</extends>
   <header>maincontent/controlwidget/listcontrol/listwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidgetControl</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
